buildozer/targets/android.py

```diff
@@ -92,6 +92,16 @@ def __init__(self, *args, **kwargs):
         if port is not None:
             self.extra_p4a_args += ' --port={}'.format(port)
 
+        activity_package = self.buildozer.config.getdefault(
+            'app', 'android.activity_package', 'org.kivy.android')
```

Comment:
Should we also document both params by updating the `default.spec`?
The idea is to comment out an example there (I would use the default values)

Reply:
yes, i will do that later, @AndreMiras  👍

Reply:
updated the PR... but now i see there is some error in the build: `Aidl not found, please install it.`   I feel pretty confident that my changes have nothing to do with that...

Reply:
Indeed, that could be a temporary glitch, like network hiccup. I'll keep an eye on the CI and restart it. Thanks for pointing it out

Reply:
Restarted the CI and still failing, it's actually because of this change I think:
https://github.com/kivy/buildozer/pull/1169/files#r446697206
Yes this is fishy that such a change break it, but it's because that line is `sed`-ed I think :sweat_smile:

Reply:
oh....
i thought it was just a mistake and removed that extra space ...

Reply:
It's definitely some inconsistencies I agree, but then some CI tweaks got built upon it :facepalm: so here we are :smile: